cmd/snap/cmd_get.go

```diff
@@ -76,14 +78,14 @@ func init() {
 	}
 
 	addCommand("get", shortGetHelp, longGetHelp, func() flags.Commander { return &cmdGet{} },
-		map[string]string{
+		waitDescs.also(map[string]string{
```

Comment:
if the get command won't support --no-wait, why have it in the args description? the `x.NoWait` check below should be enough, no?

Reply:
`snap` expects a description for each option so if it's not included here, the parser creation panics. But I agree, this will look a bit confusing. I was trying to avoid making a new mixin but I'll do that. It'll be basically the same as the waitMixin but without NoWait

Reply:
Actually, `cmdWatch` also uses waitMixin without NoWait and it was using it in a hacky way, so a new mixin makes more sense than I thought